stan/math/prim/scal/prob/chi_square_ccdf_log.hpp

```diff
@@ -93,7 +93,7 @@ namespace stan {
         const T_partials_return beta_dbl = 0.5;
 
         // Compute
```

Comment:
I reviewed the other one while watching the kids at the pool so short chunks works for me!